### PR TITLE
[super-agent] remove jobs

### PR DESCRIPTION
**Pull request by paologallinaharbur**
#### Is this a new chart
no

#### What this PR does / why we need it:

Right now we do not have auto=upgrade in place, therefore it is better to install in the dependency in an easy way.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[mychartname]`)

### CODE REVIEW

**Comment by paologallinaharbur** [comment]
The upgrade path is failing since we are introducing a breaking change and the tool is not smart enough to detect it 😄

File: charts/super-agent/README.md [hunk 1]
```diff
@@ -47,20 +47,8 @@ As of the creation of the chart, it has no particularities and this section can
...
-| helm.release.version | string | `"0.0.12-beta"` | The Helm chart of the super-agent. This values is meant to be changed only on air-gapped environments or for development/testing purposes.  TODO: Point renovatebot here. |
-| helm.repository | object | See `values.yaml` | Values related to the Helm repository where to download the super agent's chart. |
-| helm.repository.certSecretRef | string | `nil` (no secret reference) | secret of type `kubernetes.io/tls` with the standard keys `tls.crt`, `tls.key`, and `ca.crt` |
-| helm.repository.secretRef | string | `nil` (no secret reference) | A reference to a secret with the keys username and password to authenticate to the repository. |
-| helm.repository.updateInterval | string | `"24h"` | Sets the interval the repository is going to be updated on the controller. |
-| helm.repository.url | string | `"https://helm-charts.newrelic.com"` | The repository where the super-agent has the chart. This values is meant to be changed only on air-gapped environments or for development/testing purposes. |
+| super-agent-deployment | object | See `values.yaml` | Values related to the super agent's Helm chart release. |
```
Comment (marcsanmi) [inline charts/super-agent/README.md:50]: I'd bump the version in the `README.md` as well: `Version: 0.0.7-beta`

**Review by marcsanmi — APPROVED** [review]
LGTM 👍

**Review by marcsanmi — APPROVED** [review]

File: charts/super-agent/README.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 # super-agent
 
-![Version: 0.0.6-beta](https://img.shields.io/badge/Version-0.0.6--beta-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square)
+![Version: 0.0.8-beta](https://img.shields.io/badge/Version-0.0.8--beta-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square)
```
Comment (kang-makes) [inline charts/super-agent/README.md:5]: I would remove This lines o we do not have to change the version each release: https://github.com/newrelic/helm-charts/blob/c48438effaf48fbc590c7acbc3c4097f4931e770/charts/super-agent/README.md.gotmpl#L6

**Review by kang-makes — APPROVED** [review]
LGTM!

File: charts/super-agent/README.md (hunk 2 above)
Comment (paologallinaharbur) [reply charts/super-agent/README.md:5]: sure! it makes sense!

File: charts/super-agent/README.md.gotmpl [hunk 3]
```diff
@@ -3,8 +3,6 @@
 {{ template "chart.header" . }}
 {{ template "chart.deprecationWarning" . }}
 
-{{ template "chart.badgesSection" . }}
-
```
Comment (paologallinaharbur) [inline charts/super-agent/README.md.gotmpl:7]: this

**Pull request merged**